about.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="referrer" content="origin" />
+
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://fonts.googleapis.com/css?family=Poppins"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="./styles.css" />
+    <link rel="stylesheet" href="./about.css" />
+    <title>Free FullStack Webinar</title>
+  </head>
+
+  <body>
+    <div class="main">
+      <header class="header header-socialmedia">
+        <div class="max-width-header-social-media">
+          <div
+            class="topnav desktopMenu-up display-header-social-media"
+            
+          >
+            <a class="" href="#"
+              ><img src="./assets/img/fb-logo.png" alt="Facebook logo" width="8"
+            /></a>
+            <a class="" href="#"
+              ><img src="./assets/img/tw-logo.png" alt="Twitter logo" width="19"
+            /></a>
+
+            <a class="" href="#">Spanish</a>
+            <a class="" href="#">My page</a>
+            <a class="" href="#">Logout</a>
+          </div>
+        </div>
+      </header>
+
+      <header class="header">
+        <div class="max-width-header">
+          <a href="./index.html" class="logo"
+            ><img src="./assets/img/logo.png" alt="Main logo" width="100"
+          /></a>
+          <div class="desktopMenu">
+            <input
+              class="menu-btn"
+              type="checkbox"
+              title="Reset Form"
+              value="Reset Form"
+              id="menu-btn"
+            />
+            <nav>
+              <label class="menu-icon" id="menuButton" for="menu-btn">
+                <span class="navicon"></span
+              ></label>
+            </nav>
+          </div>
+
+          <div class="topnav desktopMenu-up" id="myTopnav">
+            <a class="hover_header" href="#logo-winner">Logo</a>
+            <a class="hover_header" href="#past-summit">Summit 2021</a>
+            <a class="hover_header squareButton" href="./index.html">Home</a>
+          </div>
```

Comment:
- [ ] This was pointed out in the previous review and it doesn't seem to have been addressed. I have seen you made changes to something similar to this but that is in the `index.html`, well done 👏🏼. Now come to the `about.html` file where this issue was originally pointed out and implement the same.

NOTE: To make our HTML more semantic, anything that is meant for navigation of the website must be wrapped in `nav` tags. 

```suggestion
          <nav class="topnav desktopMenu-up" id="myTopnav">
            <a class="hover_header" href="#logo-winner">Logo</a>
            <a class="hover_header" href="#past-summit">Summit 2021</a>
            <a class="hover_header squareButton" href="./index.html">Home</a>
          </nav>
```

Reply:
My bad, forgot to do it in the other file, thanks!!